Eevee.Sleep.Bot/Handlers/InteractionHandler.cs

```diff
@@ -126,14 +126,19 @@ await interaction
 
         await component.RespondAsync("You have been given the role!", ephemeral: true);
 
-        var roleId = ulong.Parse(customId.Replace("role", ""));
-        var user = (SocketGuildUser)component.User;
+        if (component.User is not null and SocketGuildUser) {
```

Comment:
This should be something `if (component.User is SocketGuildUser)` so no type casting (`(SocketGuildUser)component.User;`) is needed.

Reply:
I also believed it would work, but the compiler would not allow it

![image](https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/assets/6148777/d9991a5e-c05d-4ce3-a11f-70721e3667c8)

Reply:
Fixed!
[042bf5a](https://github.com/RaenonX-PokemonSleep/eevee-sleep-discord-bot/pull/25/commits/042bf5ae19def3a76b4fa219af2021bdffdd459e)